packages/preview/scholarly-epfl-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "scholarly-epfl-thesis"
+version = "0.1.0"
+compiler = "0.11.0"
+entrypoint = "src/lib.typ"
+repository = "https://github.com/augustebaum/epfl-thesis-typst"
+authors = ["Auguste Baum <https://github.com/augustebaum>"]
+license = "MIT"
+description = "A template for a thesis at EPFL"
+keywords = ["thesis", "university", "EPFL", "Lausanne"]
+categories = ["thesis"]
+exclude = ["example", "latex-reference", "screenshots"]
```

Comment:
I think `example` should not be excluded here, as it is the directory that is used to initialize the template.

Reply:
Thanks for your review! I misunderstood the point of `exclude` I guess. This is now corrected.

Reply:
`exclude` tells which files should not be in the archive that will be downloaded by Typst when your package is imported. It is generally a good practice to exclude example files, but in your case you also use this directory to initialize the template associated with your package, which would not work if the directory is not part of the downloaded archive.